crates/sui-framework/sources/transfer.move

```diff
@@ -15,7 +15,9 @@ module sui::transfer {
     const ESharedNonNewObject: u64 = 0;
 
     /// Transfer ownership of `obj` to `recipient`. `obj` must have the `key` attribute,
-    /// which (in turn) ensures that `obj` has a globally unique ID.
+    /// which (in turn) ensures that `obj` has a globally unique ID. Note that while while the
```

Comment:
@amnn, thoughts on wording here?

Reply:
> Note that if the `recipient` address represents an object ID, the `obj` sent will be inaccessible after the transfer

Something like this, and then we can rephrase to describe send to object behaviour when it's implemented?  Seems strange to reference a behaviour that doesn't yet exist.